### ISSUE TITLE
Run accessibility tool

### ISSUE THREAD

**Issue by tasha-m-k** [IBM/zowe-cli-cics-deploy-plugin#180]

**Comment by tasha-m-k** [comment]
Sent to Erik Cao for checking.

**Comment by tasha-m-k** [comment]
A lot of the issues the tool is raising point to the styling (lack of contrast) and missing id="" or role="" in the html. All looks like things that are usually autogenerated by DITA that aren't an option in Markdown. Things like lack of contrast are easier to fix but I'll check with Melita how essential these issues are to fix.

**Comment by tasha-m-k** [comment]
Spoke to Melita about this, usually if there are accessibility violations we need to justify them to Simon Rachman. Things like simply making text bigger and changing colour should be easier to fix than justify, so I'd suggest changing them.
